src/utils.py

```diff
@@ -0,0 +1,26 @@
+"""Kubernetes utils library."""
+
+import logging
+import os
+import random
+import string
+
+from kubernetes import client, config
+from kubernetes.client.rest import ApiException
+from kubernetes.stream import stream
+
+logger = logging.getLogger(__name__)
+
+
+def _load_kube_config():
+    # TODO: Remove this workaround when bug LP:1892255 is fixed
```

Comment:
Seems like this issue was fixed in Juju 2.8.3 ( see [LP1892255](https://bugs.launchpad.net/juju/+bug/1892255) ) Is this work around still required ?